### DISCUSSION TITLE
Permission

### DISCUSSION THREAD

**Discussion by Motta30** [Kaltharos/RPGMods#37] [Q&A]
Podrian explicar bien porfavor de como le doy permiso a los usuarios para que puedar usar el .pvp ? porque la verdad que esta demaciado mal explicado necesito el comando exacto no un comando que esta en la parte de permission no me sirve porque nose como ponerlo correctamente  y nose como solucionar eso.

**Comment by Kaltharos** [comment]
Uh... english please??

**Comment by Motta30** [comment]
> Could you please explain well how I give permission to users to
> that I can use the .pvp ? because the truth is that it is too poorly
explained
> I need the exact command not a command that is in the permission part
> It doesn't work for me because I don't know how to put it correctly and I
don't know how to solve it> Could you please explain well how I give
permission to users to
> that I can use the .pvp ? because the truth is that it is too poorly
explained
> I need the exact command not a command that is in the permission part
> It doesn't work for me because I don't know how to put it correctly and I
don't know how to solve it

**Comment by Kaltharos** [comment]
what's confusing about it?

type pvp on to turn on pvp for you, type pvp off to turn off pvp for you...

**Comment by Motta30** [comment]
no brother, look being a common user without having an admin when
using the .pvp it tells me that I do not have permission to use that
command, I need to know exactly how I have to do so that on my server
the users can use the .pvp since in the mod is super badly explained I
don't know what I have to do so they can use it

[image: image.png]
